lib/node_modules/@stdlib/stats/base/dists/weibull/cdf/benchmark/benchmark.js

```diff
@@ -66,11 +76,16 @@ bench( pkg+':factory', function benchmark( b ) {
 	lambda = 3.14;
 	k = 2.25;
 	mycdf = cdf.factory( k, lambda );
+	len = 100;
+
+	x = new Float64Array( len );
```

Comment:
```suggestion
	len = 100;
	x = new Float64Array( len );
```

No space between initializations